api/server/response_options.proto

```diff
@@ -48,3 +48,11 @@ message ResponseOptions {
   // responses 3 and 2>.
   string emit_previous_request_delta_in_response_header = 6;
 }
+
+message DynamicDelayConfiguration {
+  ResponseOptions response_options = 1;
```

Comment:
Why do we need the  response_options here? Logically I assume test-server filter is performing most of the work configured by response_option. The dynamic delay should only include the delay settings.

Reply:
As a result oneof_delay_options should be deprecated ?

Reply:
We've discussed this, and are renaming this to experimental_response_options with the idea of a refactor soon after